Disable clamav official signatures

gsanchietti opened the pull request:
If OfficialSignatures is set to `disabled`, the update event will:

- disable freshclam execution
- remove downloaded official signatures

NethServer/dev#5803

gsanchietti committed:
Disable clamav official signatures

If OfficialSignatures is set to `disabled`, the update event will:

- disable freshclam execution
- remove downloaded official signatures

gsanchietti committed:
Add README

[skip ci]

DavidePrincipi reviewed (approved):
Ok, what about merging together `nethserver-antivirus-init-avdb` and `nethserver-antivirus-officialsigs`?

gsanchietti commented:
> Ok, what about merging together `nethserver-antivirus-init-avdb` and `nethserver-antivirus-officialsigs`?

I've split the code for better readability, but I agree it could be easily merged!

gsanchietti committed:
Merge officialsigs and init-avdb actions

gsanchietti committed:
Replace unused configuration file

- clamav-unofficial-sigs.conf has been replaced by user.conf
- support rating selection, valid values for 'UnofficialSignaturesRating' prop are: 'high', 'medium', 'low'
